### PR TITLE
Core: Add update stocks signal

### PR DESCRIPTION
**Pull request by tulimaki**

### CODE REVIEW

**Comment by tulimaki** [comment]
should be ready to go

File: shuup/core/models/_product_shops.py [hunk 1]
```diff
@@ -426,6 +379,96 @@ def get_quantity_errors(self, quantity):
...
+            try:
+                child_shop_product = child_product.get_shop_instance(self.shop)
+            except ShopProduct.DoesNotExist:
+                continue
+
+            if child_shop_product.is_orderable(
+                    supplier=supplier,
+                    customer=customer,
+                    quantity=child_shop_product.minimum_purchase_quantity,
+                    allow_cache=False
```
[the remark this answers is not in the document]
Comment (tulimaki) [reply shuup/core/models/_product_shops.py]: I think I didn't change this. And I think this shouldn't be change either. The whole orderability change here should be pretty much purely just improving code.

File: shuup/core/models/_product_shops.py [hunk 2]
```diff
@@ -426,6 +379,96 @@ def get_quantity_errors(self, quantity):
                         larger_amount=render_qty(larger_p))
             yield ValidationError(message, code="invalid_purchase_multiple")
 
+    def get_supplier_errors(self, supplier, customer, quantity, ignore_minimum):
+        enabled_supplier_pks = self.suppliers.enabled().values_list("pk", flat=True)
```
[the remark this answers is not in the document]
Comment (tulimaki) [reply shuup/core/models/_product_shops.py]: why?

[the remark this answers is not in the document]
Comment (chessbr) [reply shuup/core/models/_product_shops.py]: nvm, I though the bool test below could not evaluate the real value from db, but it looks it does

**Pull request merged**